apps/web/src/components/widget/AvatarContainer.tsx

```diff
@@ -0,0 +1,194 @@
+import { useState } from 'react';
+import { Box, Group, Divider, Popover, Stack, Avatar as MAvatar } from '@mantine/core';
+import {
+  WarningFilled,
+  InfoCircleFilled,
+  CheckCircleFilled,
+  CloseCircleFilled,
+  UpCircleFilled,
+  QuestionCircleFilled,
+} from '@ant-design/icons';
+import { SystemAvatarIconEnum, IAvatarDetails, AvatarTypeEnum } from '@novu/shared';
+import { colors, Input, Switch, Text, Tooltip } from '../../design-system';
+import { Camera } from '../../design-system/icons/general/Camera';
+import { Avatar } from '../../design-system/icons/general/Avatar';
+import { AvatarWrapper, IconWrapper, useStyles } from './AvatarContainer.styles';
+
+const systemIcons = [
+  {
+    icon: <WarningFilled />,
+    type: SystemAvatarIconEnum.WARNING,
+    bgColor: '#FFF000',
+  },
+  {
+    icon: <InfoCircleFilled />,
+    type: SystemAvatarIconEnum.INFO,
+    bgColor: '#0000FF',
+  },
+  {
+    icon: <UpCircleFilled />,
+    type: SystemAvatarIconEnum.UP,
+    bgColor: colors.B70,
+  },
+  {
+    icon: <QuestionCircleFilled />,
+    type: SystemAvatarIconEnum.QUESTION,
+    bgColor: colors.B70,
+  },
+  {
+    icon: <CheckCircleFilled />,
+    type: SystemAvatarIconEnum.SUCCESS,
+    bgColor: colors.success,
+  },
+  {
+    icon: <CloseCircleFilled />,
+    type: SystemAvatarIconEnum.ERROR,
+    bgColor: colors.error,
+  },
+];
+
+const AvatarContainer = ({ value, onChange }: { onChange: (data: any) => void; value: IAvatarDetails }) => {
```

Comment:
I'm also wondering if won't be better if we will add some border around that Popover body and arrow because the backgrounds are merging together and making it a worse look. WDYT and maybe we should get the feedback from others here?

![Screenshot 2022-11-07 at 14 31 14](https://user-images.githubusercontent.com/2607232/200323570-434f33b8-555a-4ff8-a958-5a6ecf0a2e69.png)

solution:
![Screenshot 2022-11-07 at 14 28 21](https://user-images.githubusercontent.com/2607232/200322626-573aad41-7d62-474f-91a6-5f4ccea9ddbe.png)

Reply:
Tend to agree that we need to change something so that the boundaries are clearer but I'm not sure what the best way to do so.

Reply:
I played around with different placement , position for the popover,
The `right` position felt better,
wdyt? @LetItRock , @djabarovgeorge 
https://user-images.githubusercontent.com/29251776/200817464-677414b8-6166-43f9-9749-4683ba66b6b7.mov

Reply:
I like that now we can see the popover boundaries.

Reply:
much much better 👍